software/fishstick/src/lib.rs

```diff
@@ -1,4 +1,16 @@
-#![no_std]
+#![cfg_attr(not(test), no_std)]
```

Comment:
does this include the standard library when running tests?

Reply:
https://ferrous-systems.com/blog/test-embedded-app/#accessing-std-when-testing-embedded-code-testing-host2target

Note that these are the tests for `fishstick` which should be able run entirely on the host during development, but on the target for deployment. Having `std` for development (testing) is useful.